tests/charts/helm_template_generator.py

```diff
@@ -102,7 +102,7 @@ def validate_k8s_object(instance, kubernetes_version):
 
 
 def render_chart(
-    name="RELEASE-NAME",
+    name="release-name",
```

Comment:
Can you tell little more about it?

Reply:
This based on previous PR https://github.com/apache/airflow/pull/25582. I rebased in on top to see if it works.